app/frontend/src/components/base/BaseActionCard.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <router-link class="action-link" :to="{ name: linkName, params: linkParams }">
+    <v-card class="action-card" max-width="344">
+      <slot></slot>
+    </v-card>
+  </router-link>
+</template>
+
+<script>
+export default {
+  name: 'BaseActionCard',
+  props: {
+    linkName: String,
+    linkParams: Object
+  }
+};
+</script>
+
+<style scoped>
+.v-card .action-card {
+  background-color: #003366;
+  border-top: 10px;
```

Comment:
We should change this to be rem/em values if possible

Reply:
Will do in next round when styling the cards.